shacl12-profiling/index.html

```diff
@@ -804,29 +804,106 @@ <h3>Document Conventions</h3>
 
 		<section id="profiles-of-shacl">
 			<h2>Profiles of SHACL</h2>
-			<p>Content.</p>
-      <p class="todo">
-        Remove this reference test: [[RDF12-PRIMER]]
+      <p>
+        This section describes profiles of the total SHACL 1.2 specification and how to create other profiles of SHACL.
       </p>
+      <section id="base-profiles">
+        <h3>Base Profiles</h3>
+        <p>
+          Base profiles of SHACL are profiles that correspond to the individual SHACL 1.2 specifications listed in the <a href="#specifications">Specifications</a> section, above.
+        </p>
+        <p>
+          Together, these Base Profiles cover all of SHACL and, when used together in the <a href="#union-profile">Union Profile</a>, they can validate that any graph is valid SHACL 1.2 data.
+        </p>
+      </section>
+      <section id="non-base-profiles">
+        <h3>Non-base Profiles</h3>
+        <p>
+          Non-base profiles of SHACL are profiles that do not correspond directly to a SHACL 1.2 specification. These profiles include supersets of the specification-derived base profiles, such as the <a href="#union-profile">Union Profile</a> defined below, and also other profiles of parts of SHACL, such as <a href="#profile-x">Profile X</a> that contains SHACL elements that, when used for validation, require no more than XXX computational complexity.
+        </p>
+        <section id="union-profile">
+          <h4>Union Profile</h4>
+        </section>
+        <section id="profile-x">
+          <h4>Profile X</h4>
+        </section>
+      </section>
+      <section id="creating-profiles">
+        <h3>Creating other Profiles</h3>
+      </section>
 		</section>
 
 		<section id="profiling-with-shacl">
 			<h2>Profiling with SHACL</h2>
-			<p>Content.</p>
+			<p>
+        This section describes how to profile specifications with SHACL.
+      </p>
+      <section id="profile-parts">
+        <h3>Profile Part Roles</h3>
+        <p>
+          SHACL is a language that can implement constraints and inference rules for RDF data. These two things are not the total set of things that a profile designers may wish to do, for example, they may want to include documentation about why and how the profile was created or provide new model elements - classes, predicates etc. - within an extended schema.
+        </p>
+        <p>
+          Since SHACL cannot be used for all possible profile parts, profile designers must look to specifications outside the <a href="#specifications">SHACL family of specifications</a> for guidance on a profile's total set of parts and how to relate them to one another. [[[dx-prof]]] is recommended for this.
```

Comment:
> ... is recommended for this.

How much group review is required to put the word "Recommended" into this text?

Reply:
Is this whole profiling document going to frame itself as an application of dx-prof?  If so, I think we could bypass the discussion around whether the group is making any kind of recommendation.

Reply:
> How much group review is required to put the word "Recommended" into this text?

> Is this whole profiling document going to frame itself as an application of dx-prof? If so, I think we could bypass the discussion around whether the group is making any kind of recommendation.

The underlying issue for both these points is that DX PROF is only a Note, not a Rec. If it was a Rec, then of course we could/should recommend it as it would be the W3C's recommended way of working.

(I don't think there's any issue with DX PROF that prevents it from becoming a Rec as the content has proved useful and uncontroversial for a number of years, but within the Dataset Exchange WG, we encountered internal conflicts that ran out the clock on WG time to get it through the system)

So perhaps we should remove the word "Recommended" but then I'm uncertain as to how strongly all this work should be framed in relation to DX PROF. 

We can't directly update DX PROF to get it accepted as a Rec for now, so we will have to decide how strongly framed we are in relation to it. I think we have to accept notions of profiles having parts with roles to move ahead with shat SHACL can do - what roles it's performing - so if we don't just accept them directly from PROF, we will have to either re-implement the logic here or appeal to a conceptual logic from first principles that PROF could be seen as just a peer implementation of.

Reply:
"Recommended" seems outside of our ability to write here.

If this document (`shacl12-profiling`) gets revised to "Build atop" DX PROF with DX PROF as a Note, then is this document also limited to being a Note?

Reply:
Note that there is also a `must` in this paragraph. It's best to avoid both `must` and `recommended` (among other RFC2119 keywords, even when not allcaps) in NOTEs and other non-normative content. Note that I've not wrapped these lines, so GitHub should present my suggested changes in proper highlighted style.
```suggestion
          Since SHACL cannot be used for all possible profile parts, profile designers need to look to specifications outside the <a href="#specifications">SHACL family of specifications</a> for guidance on a profile's total set of parts and how to relate them to one another. [[[dx-prof]]] is expected to be used for this.
```